src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -48,41 +88,61 @@ export type Props = StyledUserDetailsTableProps & {
 
 interface RowProps {
   order: Order
-  isPriceInverted: boolean
+  _isPriceInverted: boolean
 }
 
-const RowOrder: React.FC<RowProps> = ({ order, isPriceInverted }) => {
+const RowOrder: React.FC<RowProps> = ({ order, _isPriceInverted }) => {
   const { creationDate, buyToken, buyAmount, sellToken, sellAmount, kind, partiallyFilled, shortId, uid } = order
+  const [isPriceInverted, setIsPriceInverted] = useState(_isPriceInverted)
+
+  useEffect(() => {
+    setIsPriceInverted(_isPriceInverted)
+  }, [_isPriceInverted])
+
+  const invertLimitPrice = (): void => {
+    setIsPriceInverted((previousValue) => !previousValue)
+  }
 
   return (
     <tr key={shortId}>
       <td>
-        {
-          <RowWithCopyButton
-            className="span-copybtn-wrap"
-            textToCopy={uid}
-            contentsToDisplay={
-              <Link to={`/orders/${order.uid}`} target="_blank">
-                {shortId}
-              </Link>
-            }
-          />
-        }
+        <span className="header-title">
+          Order ID <HelpTooltip tooltip={tooltip.orderID} />
+        </span>
+        <RowWithCopyButton
+          className="span-copybtn-wrap"
+          textToCopy={uid}
+          contentsToDisplay={
+            <Link to={`/orders/${order.uid}`} target="_blank">
+              {shortId}
+            </Link>
+          }
+        />
       </td>
       <td>
+        <span className="header-title">Type</span>
         <TradeOrderType kind={kind} />
       </td>
       <td>
+        <span className="header-title">Sell Amount</span>
         {formattedAmount(sellToken, sellAmount.plus(order.feeAmount))} {sellToken?.symbol}
       </td>
       <td>
+        <span className="header-title">Buy amount</span>
         {formattedAmount(buyToken, buyAmount)} {buyToken?.symbol}
       </td>
-      <td>{getLimitPrice(order, isPriceInverted)}</td>
       <td>
+        <span className="header-title">
+          Limit price <Icon icon={faExchangeAlt} onClick={invertLimitPrice} />
```

Comment:
Does this render/have an `alt=""` attribute for the image/icon?

Reply:
`FontAwesomeIcon` does not allow alt prop.